src/main/cpp/subsystems/Elevator.cpp

```diff
@@ -6,81 +6,158 @@
+            Talon::LengthToSRXUnit(height)));
       },
       [this]
       {
-        m_vertical_motor_left.Set(ctre::phoenix::motorcontrol::ControlMode::PercentOutput, 0.0);
+        // stop motor
+        m_left_motor.Set(ctre::phoenix::motorcontrol::ControlMode::PercentOutput, 0.0);
       })
-      .Until([this] { return m_vertical_motors_pid.AtSetpoint(); });
+      .Until([this] { return m_elevator_motors_pid.AtSetpoint(); });
   }
 
-  frc2::CommandPtr Elevator::MoveToLevelCommand(int)
-  {
+  frc2::CommandPtr Elevator::MoveToLevelCommand(int level)
+  {                               //  Coral Intake     Elevator height
     static const std::array<std::pair<units::degree_t, units::inch_t>, 5> presets {{
       { 85_deg, 0_in },
-      { 35_deg, 18_in - BASE_HEIGHT_FROM_FLOOR },
-      { 35_deg, 31.875_in - BASE_HEIGHT_FROM_FLOOR },
-      { 35_deg, 47.875_in - BASE_HEIGHT_FROM_FLOOR },
-      { 88_deg, 72_in - BASE_HEIGHT_FROM_FLOOR },
+      { 35_deg, 18_in - BASE_HEIGHT_FROM_FLOOR - ELEVATOR_LOWEST_POINT_FROM_BASE },
+      { 35_deg, 31.875_in - BASE_HEIGHT_FROM_FLOOR - ELEVATOR_LOWEST_POINT_FROM_BASE },
+      { 35_deg, 47.875_in - BASE_HEIGHT_FROM_FLOOR - ELEVATOR_LOWEST_POINT_FROM_BASE },
+      { 88_deg, 72_in - BASE_HEIGHT_FROM_FLOOR - ELEVATOR_LOWEST_POINT_FROM_BASE },
     }};
 
-    m_level = std::clamp(m_level, 0, static_cast <int> (presets.size()) - 1);
+    m_level = std::clamp(level, 0, static_cast <int> (presets.size()) - 1);
     
-    auto [angle, height] = presets.at(m_level);
+    auto [angle, height] = presets.at(level);
 
-    return MoveWristToCommand(angle).AndThen(ElevateToCommand(height));
+    return ElevateToCommand(height).AndThen(MoveWristToCommand(WristType::kCoral, angle));
   };
 
-
   frc2::CommandPtr Elevator::MoveUpOnceCommand()
   {
-    return this->MoveToLevelCommand(m_level + 1); 
+    m_level += 1;
+    return this->MoveToLevelCommand(m_level); 
   }
 
   frc2::CommandPtr Elevator::MoveDownOnceCommand()
   {
-    return this->MoveToLevelCommand(m_level - 1); 
+    m_level -= 1;
+    return this->MoveToLevelCommand(m_level); 
+  }
+
+  frc2::CommandPtr Elevator::MoveToRestCommand()
+  {
+    //move 18 inches from start to provide space, and then move wrist.
+    return this->ElevateToCommand(18_in).AndThen(MoveWristToCommand(WristType::kAlgae, 0_deg));
+  }
+
+  frc2::CommandPtr Elevator::MoveElevatorByPower(double val)
+  {
+    return this->RunOnce
+    (
+      [this, val]
+      {
+        m_left_motor.Set(ctre::phoenix::motorcontrol::ControlMode::PercentOutput, val);
+      }
+    );
+  }
+
+  frc2::CommandPtr Elevator::MoveAlgaeWristByPower(double val)
+  {
+      
+    return this->RunOnce
+    (
+      [this, val]
+      {
+        m_algae_wrist_motor.Set(val);
+      }
+    );
+  
+  }
+
+  frc2::CommandPtr Elevator::MoveCoralWristByPower(double val)
+  {
+      
+    return this->RunOnce
+    (
+      [this, val]
+      {
+        m_coral_wrist_motor.Set(val);
+      }
+    );
+  
```

Comment:
This is short enough to turn into a one-liner. Also, you have some unnecessary blank lines here.

```suggestion
    return this->RunOnce([this, val] { m_coral_wrist_motor.Set(val); });
```